concourse/pipelines/autodelete-cloudfoundry.yml

```diff
@@ -129,8 +129,8 @@ jobs:
           image_resource:
             type: docker-image
             source:
-              repository: governmentpaas/bosh-cli
-              tag: 465642da06051a55630d39c899697b678f66a7f7
+              repository: alpine
+              tag: "3.4"
```

Comment:
I don't think we should pin versions here. I know we do it in other places, but I think that is a mistake. Two reasons:

* We trust these standard libraries, so should be okay with whatever new versions they release
* There is maintenance involved in updating these tags
* We have no need for pinning, because we don't intend to run different versions of the same container like we do with our in-house images.